Upstream: under review, https://github.com/doxygen/doxygen/pull/6750
Reason: The '\' in front of makeindex lets cmake choke when trying to
        build documentation for cmocka and libssh

From 0c080a2491e3d4b462f7ae7ca89a52d073295964 Mon Sep 17 00:00:00 2001
From: albert-github <albert.tests@gmail.com>
Date: Mon, 7 Jan 2019 18:26:34 +0100
Subject: [PATCH] issue #6725 Doxygen 1.8.15 CMake 3.13 incompatibility

Fix to overcome problems in CMake. The required `\` in the `tex`  code is automatically added when generating the LaTeX files.
---
 src/config.xml | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/config.xml b/src/config.xml
index fdc562d1..6b61f3ce 100644
--- a/src/config.xml
+++ b/src/config.xml
@@ -2594,11 +2594,12 @@ EXTRA_SEARCH_MAPPINGS = tagname1=loc1 tagname2=loc2 ...
 ]]>
       </docs>
     </option>
-    <option type='string' id='LATEX_MAKEINDEX_CMD' defval='\makeindex' depends='GENERATE_LATEX'>
+    <option type='string' id='LATEX_MAKEINDEX_CMD' defval='makeindex' depends='GENERATE_LATEX'>
       <docs>
 <![CDATA[
  The \c LATEX_MAKEINDEX_CMD tag can be used to specify the command name to
- generate index for \f$\mbox{\LaTeX}\f$.
+ generate index for \f$\mbox{\LaTeX}\f$. In case there is no backslash (`\`) as first character it
+ will be automatically added in the \f$\mbox{\LaTeX}\f$ code.
 
  @note This tag is used in the generated output file (`.tex`).
  \sa  \ref cfg_makeindex_cmd_name "MAKEINDEX_CMD_NAME" for the part in the `Makefile` / `make.bat`.
-- 
2.20.1

